documentation/source/swagger/swagger_channels_apis.yaml

```diff
@@ -1082,62 +1082,98 @@ components:
+        additionalInfo:
+          type: string
+          maxLength: 100
+          example: "Mensagem complementar necessária para o agrupamento da identificação do telefone."
+    #### PhoneChannelService ####
+    PhoneChannelService:
+      type: object
+      required:
+        - name
+        - code
+      properties:
+        name: 
+          type: string
+          enum:
+            - ABERTURA_CONTAS_DEPOSITOS_OU_PAGAMENTO_PRE_PAGA
+            - SAQUE_MOEDA_EM_ESPECIE
+            - RECEBIMENTOS_PAGAMENTOS_QUALQUER_NATUREZA
+            - TRANSFERENCIAS_ELETRONICAS_VISANDO_MOVIMENTACAO_CONTAS_DEPOSITOS_OU_PAGAMENTO_TITULARIDADE_CLIENTES
+            - CONSULTA_SALDOS_EXTRATOS_CONTAS_DEPOSITOS_CONTAS_PAGAMENTOS
+            - APLICACOES_RESGATES_INVESTIMENTOS
+            - EXECUCAO_ATIVA_PASSIVA_ORDENS_PAGAMENTO_SOLICITACAO_CLIENTES_USUARIOS
+            - DEPOSITOS_MOEDA_ESPECIE_CHEQUE
+            - OPERACOES_CREDITO_BEM_COMO_OUTROS_SERVICOS_PRESTADOS_ACOMPANHAMENTO_OPERACAO
+            - CARTAO_CREDITO
+            - SEGUROS
+            - OPERACOES_ARRENDAMENTO_MERCANTIL
+            - ABERTURA_CONTA_PAGAMENTO_POS_PAGA
+            - COMPRA_VENDA_MOEDA_ESTRANGEIRA_ESPECIE
+            - COMPRA_VENDA_CHEQUE_CHEQUE_VIAGEM_BEM_COMO_CARGA_MOEDA_ESTRANGEIRA_CARTAO_PRE_PAGO
+            - COMPRA_VENDA_OURO
+            - OUTROS_PRODUTOS_SERVICOS
+            - CANCELAMENTO
+            - INFORMACOES
+            - RECLAMACOES
+          description: "Nome dos Serviços efetivamente prestados pelo Canal de Atendimento"
+        code:
+          type: string
+          enum:
+            - ABRE_CONTA_DEPOSITO_OU_PRE_PAGA
+            - SAQUE_MOEDA_ESPECIE
+            - RECEBE_PAGA_QUALQUER_NATUREZA
+            - TRANSFERENCIAS_ELETRONICAS_MOVIMENTA_CONTAS_DEPOSITOS_OU_PAGTO_TITULARES_CLIENTES
+            - CONSULTA_SALDOS_EXTRATOS_CONTAS_DEPOSITOS_PAGTOS
+            - APLICA_RESGATA_INVESTIMENTOS
+            - EXECUCAO_ATIVA_PASSIVA_ORDENS_PAGTO
+            - DEPOSITO_MOEDA_ESPECIE_CHEQUE
+            - OPERA_CREDITO_OUTROS_SERVICOS_ACOMPANHA_OPERACAO
+            - CARTAO_CREDITO
+            - SEGUROS
+            - OPERA_ARRENDAMENTO_MERCANTIL
+            - ABERTURA_CONTA_PAGAMENTO_POS_PAGA
+            - COMPRA_VENDA_MOEDA_ESTRANGEIRA_ESPECIE
+            - COMPRA_VENDA_CHEQUE_CHEQUE_VIAGEM_CARGA_MOEDA_ESTRANGEIRA_CARTAO_PRE_PAGO
+            - COMPRA_VENDA_OURO
+            - OUTROS_PRODUTOS_SERVICOS
+            - CANCELAMENTO
+            - INFORMACOES
+            - RECLAMACOES
+          description: "Código dos Serviços efetivamente prestados pelo Canal de Atendimento"
         additionalInfo:
           type: string
           maxLength: 2000
-          description: "Campo de texto livre para descrever mais sobre os serviços"
+          description: "Texto livre para complementar informação relativa ao Serviço disponível, quando for selecionada a opção p preenchida a opção 'OUTROS_PRODUTOS_SERVICOS'"
```

Comment:
A seguinte frase está com erro de grafia e redundância.

`for selecionada a opção p preenchida a opção 'OUTROS_PRODUTOS_SERVICOS'`

Reply:
Será corrigido após ver qual é o texto correto com a Marili, porque está dessa forma no dicionário.